js/test.js

```diff
@@ -0,0 +1,81 @@
+import { getQuestions } from './util.js';
+
+// Get references to the HTML elements
+const mainContainer = document.getElementById('mainContainer')
+const questionNumberElement = document.getElementById('questionNumber');
+const questionTextElement = document.getElementById('questionText');
+const button1 = document.getElementById('button1');
+const button2 = document.getElementById('button2');
+const loading = document.getElementById("loading");
+const ad = document.getElementById("ad");
+
+// Sample questions and question numbers
+var test_id = null
+var lan = null
+var questions = []
+var currentQuestionIndex = 0;
+
+// Function to update the question and question number
+function updateQuestion() {
+    if (currentQuestionIndex < questions.length) {
+        const currentQuestion = questions[currentQuestionIndex];
+        questionNumberElement.textContent = `Question ${currentQuestionIndex + 1}:`;
+        questionTextElement.textContent = currentQuestion.question;
+        
+        // Update answer buttons
+        button1.textContent = currentQuestion.answers[0];
+        button2.textContent = currentQuestion.answers[1];
```

Comment:
Are we not going to randomize the order of answers?

Reply:
Move on for now and make PR later.